rewrites.js

```diff
@@ -126,6 +126,14 @@ exports.REWRITES = [
     source: '/:parentCollectiveSlug/projects/(new|create)',
     destination: '/create-project',
   },
+  {
+    source: '/dashboard',
+    destination: '/dashboard',
+  },
+  {
+    source: '/dashboard/:section?/:slug/:subpath*',
```

Comment:
IMO, `/dashboard/:slug/:section?/:subpath*` is more intuitive.

Reply:
Yeah, I think I agree. I was a bit back and forth with this. 

The idea with putting the slug last was that the AccountSwitcher might eventually be optional and more of a filter, i.e. you could go to `/dashboard/expenses` and see all expenses that concern me regardless of which Collective they belong to.

But since that is just an idea at this stage, I'm inclined to change it to what you suggest.